### ISSUE TITLE
Throw an error if unexpected HTTP status code is returned

### ISSUE THREAD

**Issue by physikerwelt** [zbMATHOpen/Update_Links#8]
https://github.com/zbMATHOpen/Update_Links/blob/e4203575f53f8b32391b4c9bbffd988faef3b985/src/update_zblinks_api/update_with_api.py#L74-L74

Currently the response is not used at all.

**Comment by dehsani-academic** [comment]
If there are thousands of entries, but only one error, I wouldn't want an error thrown.  The API will return error codes one can use to check the success or failure of entries.  Maybe a success/failure report could be created instead.

**Comment by physikerwelt** [comment]
I agree. However the method should throw an error which should be caught in the for loop.
